engines/config-query-sparql/config/expression-evaluator/actors.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^2.0.0/components/context.jsonld"
+  ],
+  "import": [
+    "ccqs:config/expression-evaluator/aggregators/average.json",
```

Comment:
Okey :D 
How about this line: 
> Each sub-directory represents a bus

It would not hold for the `expression-evaluator` directory? (I still need the `evaluators` file)

Reply:
> It would not hold for the expression-evaluator directory? (I still need the evaluators file)

Indeed, our exp-eval is not a bus, but just a component, so that's a bit of a special one.
So having a dedicated dir for that does make sense.
However, I do wonder if we should consider bus-ifying that one as well.

Reply:
I don't think that would make a lot of sense, because we decided to have our function-actors use the helpers provided by the EE (think OverloadTree).

Reply:
> So having a dedicated dir for that does make sense.

Should I edit the rules described in:  https://github.com/comunica/comunica/tree/master/engines/config-query-sparql/config#directory-structure

Reply:
(That being said, I did already make an interface for the EE :) )